python/lsst/ts/externalscripts/auxtel/latiss_cwfs_align.py

```diff
@@ -588,9 +595,9 @@ def get_schema(cls):
                 type: number
                 default: 0.8
               dataPath:
-                description: Path to the butler data repository.
+                description: Path to the butler data repository on the summit
```

Comment:
Maybe mention that this is gen3? Also is it "on the summit" or "at the summit"?

Reply:
I'd say "on" .... although I think either works...